### ISSUE TITLE
Ynet ETH/UCY config file

### ISSUE THREAD

**Issue by mulplue** [HarshayuGirase/PECNet#55]
Hello!
Can you produce the config file of ETH/UCY? I've tried SDD trajnet and longterm but I don't know the exact settings of ETH/UCY dataset. Can you release the config and training script of ETH/UCY like SDD? Thanks very much!

**Comment by SunshineDHW** [comment]
@mulplue !Hello,do know how to train and test on ETH/UCY datasets?

**Comment by mulplue** [comment]
> @mulplue !Hello,do know how to train and test on ETH/UCY datasets?

Yes, I have tried to reproduce on ETH/UCY benchmark with similiar config in SDD. But my reproduce points are higher than the reported points. Here is a [reproduce report](https://openreview.net/pdf?id=HV2zgpM7n0F), I get similiar but slightly lower results of this report so you may refer to this.

**Comment by SunshineDHW** [comment]
> > @mulplue !Hello,do know how to train and test on ETH/UCY datasets?
> 
> Yes, I have tried to reproduce on ETH/UCY benchmark with similiar config in SDD. But my reproduce points are higher than the reported points. Here is a [reproduce report](https://openreview.net/pdf?id=HV2zgpM7n0F), I get similiar but slightly lower results of this report so you may refer to this.

Thank you very much!

**Comment by githubpppppp** [comment]
> > @mulplue !Hello,do know how to train and test on ETH/UCY datasets?
> Yes, I have tried to reproduce on ETH/UCY benchmark with similiar config in SDD. But my reproduce points are higher than the reported points. Here is a [reproduce report](https://openreview.net/pdf?id=HV2zgpM7n0F), I get similiar but slightly lower results of this report so you may refer to this.

@mulplue !Hello, can you upload the overall code of your report for download and reference? thank you very much!

**Comment by mulplue** [comment]
> > > @mulplue !Hello,do know how to train and test on ETH/UCY datasets?
> > > Yes, I have tried to reproduce on ETH/UCY benchmark with similiar config in SDD. But my reproduce points are higher than the reported points. Here is a [reproduce report](https://openreview.net/pdf?id=HV2zgpM7n0F), I get similiar but slightly lower results of this report so you may refer to this.
> 
> @mulplue !Hello, can you upload the overall code of your report for download and reference? thank you very much!

Sorry, I lost the code due to computing resource migration... You may refer to this report https://openreview.net/pdf?id=HV2zgpM7n0F, I adopted a very similar config with it.

**Comment by githubpppppp** [comment]
> > > > @mulplue !Hello,do know how to train and test on ETH/UCY datasets?!你好，知道如何在ETH/UCY数据集上进行训练和测试吗？
> > > > Yes, I have tried to reproduce on ETH/UCY benchmark with similiar config in SDD. But my reproduce points are higher than the reported points. Here is a [reproduce report](https://openreview.net/pdf?id=HV2zgpM7n0F), I get similiar but slightly lower results of this report so you may refer to this.是的，我已经尝试在SDD中使用类似的配置在ETH/UCY基准上进行复制。但我的复制点比报道的点要高。这是一份[复制报告](https://openreview.net/pdf?id=HV2zgpM7n0F)，我得到了类似的结果，但略低，所以你可以参考这份报告。
> > 
> > 
> > @mulplue !Hello, can you upload the overall code of your report for download and reference? thank you very much!!你好，你能上传你的报告的整体代码供下载和参考吗？非常感谢！
> 
> Sorry, I lost the code due to computing resource migration... You may refer to this report https://openreview.net/pdf?id=HV2zgpM7n0F, I adopted a very similar config with it.对不起，由于计算资源迁移，我丢失了代码...您可以参考本报告[https://openreview.net/pdf? id=HV2zgpM7n0F](https://openreview.net/pdf?id=HV2zgpM7n0F)，我采用了一个非常相似的配置。

Thank you very much!
